docs/contributor/deploy-access-kubernetes.md

```diff
@@ -0,0 +1,92 @@
+# Deploying and Accessing Busola in the Kubernetes Cluster
+
+## Architecture
+
+![Busola Kubernetes architecture](assets/busola_kubernetes_3.svg)
+
+To expose Busola, you can use [APIRule](https://github.com/kyma-project/busola/tree/main/resources/istio), [Ingress](https://github.com/kyma-project/busola/tree/main/resources/ingress) or create your own exposing mechanism.
+For more details about environment configuration, see [Environment-Specific Settings](configuration.md#environment-specific-settings).
+
+## Deploying Busola in the Kubernetes Cluster
+
+To install Busola from release in the Kubernetes cluster set `VERSION` shell environment variable with desired release and run:
+
+```shell
+kubectl apply -f https://github.com/kyma-project/busola/releases/download/${VERSION}/busola.yaml
```

Comment:
I would add a note around this codeblock about namespaces. We don't want our readers to accidentally use the default namespace, as that is very bad practice. And the manifest stream in the yaml is namespace-less.